jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/oracle/OracleSqlGenerationVisitor.java

```diff
@@ -267,10 +252,11 @@ public String visit(final SqlFunctionAggregateGroupConcat function) throws Adapt
     @Override
     public String visit(final SqlFunctionAggregate function) throws AdapterException {
         String sql = super.visit(function);
-        final boolean isDirectlyInSelectList = (function.hasParent() && function.getParent().getType() == SqlNodeType.SELECT_LIST);
+        final boolean isDirectlyInSelectList = (function.hasParent()
+                && (function.getParent().getType() == SqlNodeType.SELECT_LIST));
         if (isDirectlyInSelectList && this.aggregateFunctionsCast.contains(function.getFunction())) {
             // Cast to FLOAT because result set metadata has precision = 0, scale = 0
```

Comment:
Remove the confusing comment.

Reply:
This is code I took over. Will be replaced by `sql-statment-builder` anyway soon.